src/pages/address/fragments/nfts/AddressNFTS.tsx

```diff
@@ -30,6 +38,24 @@ const AddressNFTS: React.FC<Props> = props => {
     setPage(lastState => lastState + 1)
   }
 
+  function handleNavigate(id: string) {
+    history.push(`${ROUTES.NFT.url}/${chain}/${network}/${hash}/${id}`)
+  }
+
+  useLayoutEffect(() => {
+    const screen = document.querySelector('html')
+
+    const observer = new ResizeObserver(([entry]) => {
+      if (entry.contentRect.width < 752) {
+        setToggleTypeSelected('list')
+      }
+    })
+
+    if (screen) {
+      observer.observe(screen)
+    }
+  }, [])
```

Comment:
What do you think about changing it in a hook?
I believe we don't have anything to verify if it's mobile or not.